### PR TITLE
cmd/snap-bootstrap: tweak recovery trigger log messages

### PR DESCRIPTION
**Pull request by bboozzoo**
The messages logged from recovery chooser trigger are slightly confusing. Tweak
them a little to make it more obvious.

### CODE REVIEW

File: cmd/snap-bootstrap/triggerwatch/evdev.go [hunk 1]
```diff
@@ -88,7 +88,7 @@ func (e *evdevKeyboardInputDevice) probeKeyState() (bool, error) {
 }
 
 func (e *evdevKeyboardInputDevice) WaitForTrigger(ch chan keyEvent) {
-	logger.Noticef("%s: starting wait", e)
+	logger.Noticef("%s: starting wait, %s hold to trigger", e, holdToTrigger)
```
Comment (pedronis) [inline cmd/snap-bootstrap/triggerwatch/evdev.go:91]: "2s hold will trigger"? "2s hold time to trigger" ?

Comment (anonymouse64) [reply cmd/snap-bootstrap/triggerwatch/evdev.go:91]:
```suggestion
	logger.Noticef("%s: starting wait, hold %s to trigger", e, holdToTrigger)
```

Comment (anonymouse64) [reply cmd/snap-bootstrap/triggerwatch/evdev.go:91]: I suggested "hold 2s to trigger"

Comment (pedronis) [reply cmd/snap-bootstrap/triggerwatch/evdev.go]: @bboozzoo I'm a bit confused, are these debugging messages or instructions for the user?

Comment (bboozzoo) [reply cmd/snap-bootstrap/triggerwatch/evdev.go]: Discussed on IRC with @pedronis

**Pull request merged**